mmv1/provider/terraform_validator.rb

```diff
@@ -168,7 +168,9 @@ def compile_common_files(output_folder, products, _common_compile_file)
                           ['converters/google/resources/metadata.go',
                            'third_party/terraform/utils/metadata.go.erb'],
                           ['converters/google/resources/compute_instance.go',
-                           'third_party/validator/compute_instance.go.erb']
+                           'third_party/validator/compute_instance.go.erb'],
+                          ['converters/google/resources/gkeonprem_operation.go',
```

Comment:
You should be able to eliminate this line, now! Validator's not generated at private. If that doesn't work, it'll need to point to `gkeonprem_operation.go.erb` rather than `gkeonprem_operation.go`

Reply:
Removing it altogether seemed to work. Thanks!